modules/olm-certificates.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificate-types-descriptions.adoc
+
+[id="olm-certificates_{context}"]
+= OLM certificates
+
+[discrete]
+== Management
+
+All certificates for OpenShift Lifecycle Manager (OLM) components
+(`olm-operator`, `catalog-operator`, `packageserver`, and
+`marketplace-operator`) are managed by the system.
+
+Operators installed via OLM can have certificates generated for them if they are
+providing API services. `packageserver` is one example.
+
+Certificates in the `openshift-operator-lifecycle-manager` namespace are managed
+by OLM with the exception of certificates used by Operators that require a
+validating or mutating webhook.
```

Comment:
I am unsure if this is necessary to call out, but OLM will not update the certificates of operators that it manages in proxy environments. These certificates must be managed by the user via the subscription config.

Reply:
As a user, the existence of a feature called the “cluster-wide proxy” makes an unambiguous promise that components managed by the cluster will use those settings. Cluster-wide proxy settings include a field for trust settings.

So this isn’t a minor point. Anything that breaks the promise made by calling it the settings “cluster-wide” should be fixed. If it’s not going to work as advertised it should be called out as clearly as possible to help set users’ expectations.

Or rename the feature to better indicate purpose. As it stands very few features that are documented as being part of an OpenShift Container Platform cluster actually follow these settings.

Reply:
As a user, the existence of a feature called the “cluster-wide proxy” makes an unambiguous promise that components managed by the cluster will use those settings. Cluster-wide proxy settings include a field for trust settings.

So this isn’t a minor point. Anything that breaks the promise made by calling it the settings “cluster-wide” should be fixed. If it’s not going to work as advertised it should be called out as clearly as possible to help set users’ expectations.

Or rename the feature to better indicate purpose. As it stands very few features that are documented as being part of an OpenShift Container Platform cluster actually follow these settings.